pxtblocks/plugins/flyout/verticalFlyout.ts

```diff
@@ -290,7 +290,10 @@ class CachedFlyout extends Blockly.VerticalFlyout {
             ),
         );
 
-        this.buttons_.push(button);
+        // TODO: This whole function is no longer called due to
```

Comment:
Highlighting todo. `initFlyoutButton_` no longer exists in Flyout so is no longer called. Simply commented out line of code to build without errors for now.